docker/Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH=
 # Install openssl for FIPS support
 #@follow_tag(registry.redhat.io/ubi9/ubi-minimal:latest)
 FROM registry.access.redhat.com/ubi9-minimal:9.3-1475 AS runtime
```

Comment:
you probably want to update the `FROM` images to the latest here too

Reply:
true but at some point 9.4 will be out and this might NOT work as intended. Would you prefer to just pull from `latest` instead of a pinned `:9.3-zzz` tag?

Reply:
what if we enabled renovate on this repo? https://github.com/janus-idp/operator/pull/216

Reply:
It should update minor versions, I've tested this before.  Here's an example:
Our dockerfile specified an [8.6-751](https://github.com/redhat-appstudio/application-service/actions/runs/7036061061/job/19147855994#step:3:31) version which was then upgraded to [8.7](https://github.com/redhat-appstudio/application-service/actions/runs/7036061061/job/19147855994#step:3:84).  

I agree with having renovate update the images as well just so we are covered on all aspects.  You can also configure Renovate to update EOL images